web-console/knikubevirt/vm-templates/vm-templates.md

```diff
@@ -0,0 +1,86 @@
+# Virtual machine templates
+
+### Summary
+
+- VM-templates is located at the main side menu, in 'Workloads' under 'virtual machines'.
+- VM-templates displays a list view similar to VM list with minor changes in columns and actions.
+- From this List, the user can create a new template, create a virtual machine from a selected template, edit a selected template, or delete a selected template.
+- Clicking on a template will bring the user to the template’s Overview tab (TBD).
+Creating or editing a template will display a wizard similar to the “No Template state” wizard from Create VM but with slight adjustments.
+- Users can also create templates based on existing VMs from the actions menu of any VM within the VM List View.
+
+
+
+## Template list view
+
+![Offline Template List View](img/1-0-vm-template-list.jpg)
+
+The user will be able to view a list of all templates that exist. From this List, the user can create a new template, create a virtual machine from a selected template, edit a selected template, or delete a selected template.
+
+
+
+
+## Create template from primary button
+
+
+![Offline Template List View](img/1-1-0-create-template.jpg)
+
+Clicking “Create Template” will display a wizard similar to the “No Template state” wizard from Create VM but with slight adjustments.
+
+![Offline Template List View](img/1-1-1-create-template-modal.jpg)
+
+1. Modal title changes to “Create Template”
+2. “Template” is no longer a Provision Source option
```

Comment:
This design has been updated to show Templates as a separate dropdown in the create a VM wizard. This could read: "Choosing a template from a dropdown is not available" or something like that.